src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -245,18 +245,18 @@ <h3 id="history">
                   data-toggle-button-selector="#force_disable_addon"
                   id="force_enable_addon" type="button">{{ _('Force-enable add-on') }}</button>
         </li>
-
-        {% if not addon.block %}
         <li>
+        {% if addon.blocksubmission %}
+          <a href="{{ url('admin:blocklist_blocksubmission_change', addon.blocksubmission.id) }}" class="button"
+                  id="edit_addon_blocksubmission" type="button">{{ _('View Block Submission') }}</a>
+        {% elif not addon.block %}
           <a href="{{ url('admin:blocklist_blocksubmission_add') }}?guids={{ addon.guid }}" class="button"
                   id="block_addon" type="button">{{ _('Block add-on') }}</a>
-        </li>
         {% else %}
-        <li>
           <a href="{{ url('admin:blocklist_block_change', addon.block.id) }}" class="button"
```

Comment:
Just clarifying here: This last case would occur when `addon.block = True` and ` addon.blocksubmission = False`. Does that mean that a block exists for the addon, but that a pending submission does not exist?

Reply:
Yes.  (`Addon.block` and `Addon.blocksubmission` actually return the instance or `None` but in python that's truthy or falsey)